### PR TITLE
Disable rcache by default

### PR DESCRIPTION
**Pull request by nirbhayc**
Signed-off-by: Nirbhay Choubey <nchoubey@bloomberg.net>

### CODE REVIEW

**Review by roborivers — COMMENTED** [review]
Lint check:

```diff


@@ db/db_tunables.h @@
+ Please update the tunable documentation (docs/pages/config/config_files.md) accordingly.
```

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>2/430 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>socksql_master_swings<br>recover_ddlk_sp</p>

**Review by aakash10292 — APPROVED** [review]
good to merge

File: db/db_tunables.h [hunk 1]
```diff
@@ -987,9 +988,9 @@ REGISTER_TUNABLE("random_lock_release_interval", NULL, TUNABLE_INTEGER,
                  NULL);
 REGISTER_TUNABLE("rangextlim", NULL, TUNABLE_INTEGER, &gbl_rangextunit,
                  READONLY | NOZERO, NULL, NULL, NULL, NULL);
-REGISTER_TUNABLE(
-    "rcache", "Keep a lookaside cache of root pages for B-trees. (Default: on)",
-    TUNABLE_BOOLEAN, &gbl_rcache, READONLY | NOARG, NULL, NULL, NULL, NULL);
+REGISTER_TUNABLE("rcache", "Keep a lookaside cache of root pages for B-trees. (Default: on)",
```
Comment (mponomar) [inline db/db_tunables.h:991]: Default: off?

**Review by roborivers — COMMENTED** [review]
Lint check:

```diff


@@ db/db_tunables.h @@
+ Please update the tunable documentation (docs/pages/config/config_files.md) accordingly.
```

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>2/429 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>socksql_master_swings<br>lostwrite</p>

**Comment by akshatsikarwar** [comment]
Disabled by a prior PR

**Pull request closed without merging**